examples/basics/plotting/grid_holed.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
++-----------+---+
+|    1      | 2 |
+|           |   |
+|---+-------+   |
+|   | Empty |   |
+| 3 +-------+---+
+|   |      4    |
++---+-----------+
```

Comment:
Nope, this one is broken. I'll have to fix it. I was focusing on getting "actual" plotting working